internal/sdkprovider/service/kafkatopic/kafka_topic_exists.go

```diff
@@ -0,0 +1,40 @@
+package kafkatopic
+
+import (
+	"context"
+	"log"
+	"sync"
+
+	"github.com/aiven/aiven-go-client/v2"
+	"golang.org/x/exp/slices"
+)
+
+var onceForService sync.Map
```

Comment:
consider renaming the variable to be more descriptive, like `onceCheckTopicForService` or something similar